runtime.md

```diff
@@ -129,18 +129,19 @@ This operation MUST run the user-specified program as specified by [`process`](c
 This operation MUST generate an error if `process` was not set.
 
 ### <a name="runtimeKill" />Kill
-`kill <container-id> <signal>`
+`kill <container-id> [-a,--all] <signal>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
-Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
-This operation MUST send the specified signal to the container process.
+This operation MUST send the specified signal to the container process(without `-a`) or to all processes in the container(with `-a`).
```

Comment:
As per https://github.com/opencontainers/runc/issues/3864, I think that `-a` should be removed.

In short, the runtime should already know how to kill a container:
 - if the container has own private PID namespace, kill the init;
 - if the container shares the PID namespace, kill all processes in the container's cgroup;
 - the container which does have neither the cgroup nor private PID namespace is broken and should not be created in the first place.
 
Option `-a` was added as a hack to kill the container which does not have own private PID namespace. Since the runtime already knows about it, there is no need for `-a`.

Reply:
> I think that `-a` should be removed.

I agree this when the signal is `SIGKILL`, but for signals, which is not `SIGKILL`, can't be send to all processes in the container like before.

After remove `-a`, we must treat `SIGKILL` as a special signal, it looks like unreasonable. Maybe all signals should be equal。

Reply:
> we must treat `SIGKILL` as a special signal

If we agree to remove `-a`, I think we should clarify this special signal `SIGKILL` for private pid ns container here.
So, do you have a strong opinion to remove `-a` in `kill` operation?